db/projects/nano/project.yaml

```diff
@@ -0,0 +1,83 @@
+name: Nano
+web:
+  - 'https://nano.org'
+start_date: '2016-02-29'
+description: 'Nano is a peer-to-peer cryptocurrency that employs a block-lattice infrastructure in a directed acyclic graph (DAG) algorithm. Nano coin is focused on being an instant, zero-fee and scalable currency.'
+ledgers:
+  - id: nano
+    name: Nano
+    type: blockchain
+    networks:
+      - id: main
+        type: main
+        name: Block-Lattice
+        proof_type: DPoS
+        mineable: false
+        algorithm: Blake2b
+        target_block_time: 0
+        tools:
+          - type: explorer
+            url: https://www.nanode.co/
+          - type: explorer
+            url: https://nanocrawler.cc/
+          - type: explorer
+            url: https://nanowat.ch/
+assets:
+  - id: nano
+    type: native
+    ledger: nano
+    name: Nano
+    symbol: NANO
+    max_supply: 133248290
+    webids:
+      coinmarketcap: nano
+      cryptocompare: nano
+webids:
+  email: press@nano.co
+  linkedin/company: nano-foundation
+  facebook: nanofoundation
+  twitter: nano
+  github: nanocurrency
+  reddit: nanocurrency
+  youtube/channel: UCFxXo3z9k4yvxSboEEjXO2w
+  medium: nanocurrency
+  discord: JphbBas
+  coinmarketcap: nano
+  cryptocompare: nano
+  bitcointalk: '1381323.0'
+  bitcoinwiki: Nano
+team:
+  - name: Colin LeMahieu
+    role: CEO/Founder
+    photo: https://nano.org/assets/photos/people-colin.jpg
+    webids:
+      linkedin: clemahieu
+      twitter: colinlemahieu
+  - name: Russel Waters
+    role: Developer
+    photo: https://nano.org/assets/photos/people-russel.jpg
+    webids:
+      linkedin: russel-waters-9606354b
+      twitter: argakiig
+  - name: Roy Keene
+    role: Developer
+    photo: https://avatars0.githubusercontent.com/u/5068442?s=460&v=4
+    webids:
+      github: rkeene
+whitepapers:
+  - title: 'Nano: A Feeless Distributed Cryptocurrency Network'
+    url: https://nano.org/en/whitepaper
+tags:
```

Comment:
I am not hundred percent sure, but I think that capital letters are not allowed. So change it to:
  - nofees
  - dag
  - sidechains
  - raiblocks
  - instanttransaction

or Schema could be updated

Reply:
OK, it seems that the first character can be capital, but rest has to be lower cases only:
https://github.com/checkstyle/checkstyle/issues/5504
Again, the Scheme could be change to "^[a-z][a-zA-Z0-9]*$"
We can create Issue